src/jit/gcencode.cpp

```diff
@@ -104,18 +187,20 @@ static void regenLog(unsigned encoding, InfoHdr* header, InfoHdr* state)
 
     fprintf(logFile, "InfoHdr( %2d, %2d, %1d, %1d, %1d,"
                      " %1d, %1d, %1d, %1d, %1d,"
-                     " %1d, %1d, %1d, %1d, %1d,"
-                     " %1d, %2d, %2d, %2d, %2d,"
-                     " %2d, %2d), \n",
+                     " %1d, %1d, %1d, %1d, %1d, %1d,"
```

Comment:
The number of arguments in the format string and the number of actual value arguments doesn't match - there are 23 of them in the formatting string, but 28 actual arguments.